src/libmongoc/tests/test-mongoc-uri.c

```diff
@@ -173,25 +173,6 @@ test_mongoc_uri_new (void)
    ASSERT (!bson_iter_next (&iter));
```

Comment:
Should the tests in this file be replaced with ones for `MONGOC_URI_SECONDARY_OK`?

Reply:
This was removed entirely as per CDRIVER-3989; the option has been deprecated and users should use `readPreference=secondaryPreferred`, hence removing all tests for the legacy secondaryOk URI option.